sktime/networks/lstmfcn.py

```diff
@@ -0,0 +1,127 @@
+# -*- coding: utf-8 -*-
+"""LongShort Term Memory Fully Convolutional Network (LSTM-FCN)."""
+
+__author__ = "Jack Russon"
```

Comment:
please use your GitHub ID

Reply:
Jack Russon is the person who originally implemented the network over in sktime-dl.

Is it alright if I extend the authors to `__author__ = ["Jack Russon", "solen0id"]`? It wouldn't feel right to just put my name on his work

Reply:
oh, apologies, I thought you were Jack Russon.

In that case, replace Jack Russon by his GitHub ID `jnrusson1` (and check whether he is in the contributor file)

Reply:
All done, added `jnrusson1` and myself to the `.all-contributorsrc` file :+1: